app/controllers/feedbacks_controller.rb

```diff
@@ -0,0 +1,46 @@
+class FeedbacksController < ApplicationController
+  def like
+    @did = params[:design_id]
+    retutn unless current_user.activation
+      feedback= current_user.feedback(@did)
+      if feedback.nil?
+        feedback = Feedback.new(user_id: current_user.id, design_id: params[:design_id],like: true)
+        feedback.save!
+      else
+        if feedback.like == true
+          feedback.update!(like: false)
+        else
+          feedback.update!(like: true)
+        end        
```

Comment:
Only one line can replace this 4 lines. `feedback.update_attributes(like: feedback.like)`